flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -319,40 +425,77 @@ class FlxGamepad implements IFlxDestroyable
 	}
 	
 	/**
-	 * Gets the value of the specified axis - use this only for things like
-	 * XboxButtonID.LEFT_TRIGGER, use getXAxis() / getYAxis() for analog sticks!
+	 * Gets the value of the specified axis using the "universal" ButtonID - 
+	 * use this only for things like <code>ButtonID.LEFT_TRIGGER</code>, 
+	 * use getXAxis() / getYAxis() for analog sticks!
 	 */
-	public inline function getAxis(AxisID:Int):Float
+	public inline function getAxis(AxisButtonID:ButtonID):Float
 	{
-		var axisValue = getAxisValue(AxisID);
+		return getAxisRaw(rawID(AxisButtonID));
+	}
+	
+	/**
+	 * Gets the value of the specified axis using the raw ID - 
+	 * use this only for things like <code>XboxButtonID.LEFT_TRIGGER</code>,
+	 * use getXAxis() / getYAxis() for analog sticks!
+	 */
+	public inline function getAxisRaw(RawAxisID:Int):Float
+	{
+		var axisValue = getAxisValue(RawAxisID);
 		if (Math.abs(axisValue) > deadZone)
 		{
+			#if (!flash && !next)
+				//(-1,1) range, normalize to (0,1) for legacy target only
```

Comment:
Ah, the comment confused me. Might just be me, but the way it's worded it could be taken as "the range is (always) (-1, 1), but we're only normalizing in legacy for some reason. :)